src/profile-logic/process-profile.js

```diff
@@ -1578,7 +1583,7 @@ export function processGeckoProfile(geckoProfile: GeckoProfile): Profile {
       const jsTracerThread = convertJsTracerToThread(
         thread,
         jsTracer,
-        meta.categories
+        meta.categories || []
```

Comment:
This was just to silence flow errors. If I don't add the ProfileMeta type annotation on `const meta: ProfileMeta`, flow fails when I assign values to `meta.profilingStartTime =` because the profilingStartTime field doesn't exist on the literal object 'meta'. With the meta object annotated to 'ProfileMeta', flow is unhappy on this line because the categories field is optional in ProfileMeta, but convertJsTracerToThread won't take undefined.
Given that I'm not actually changing the behavior and that afaik jsTracer is dead code at this point, I thought this way of silencing the error was fine. Do you see another nicer way?

Reply:
I see :)
You can use `geckoProfile.meta.categories` directly here, and Flow will be happy.